docs/docs/components/guidelines/README.md

```diff
@@ -131,20 +131,16 @@ run the following commands at the root of the project:
 
 To create and submit a new or updated component to the Pipedream registry:
 
-1. Fork or branch the `pipedreamhq/pipedream` repo
-2. Develop and test a new or updated component that conforms to the guidelines
-   in the document
-3. Create a PR for the Pipedream team to review and post a message in our
-   [community forum](https://pipedream.com/community/c/dev/11)
-4. Address feedback provided by Pipedream
-5. Once the review is complete and approved, Pipedream will merge the PR to the
-   `master` branch
-
-Looking for ideas? Check out
```

Comment:
Maybe keep this below the list, as its own section, like it reads here. We actually get a lot of people who peruse these lists for components to build!

Reply:
oh yea oops, def not intentional!